modules/local/predict_epitopes.nf

```diff
@@ -2,8 +2,7 @@ process PREDICT_EPITOPES {
     label 'process_low'
     label 'cache_lenient'
 
-    // TODO: conda
-    conda (params.enable_conda ? { exit 1 "Conda is currently not available for metapep" } : null)
+    conda null
```

Comment:
For ampliseq I followed the cellranger example, see https://github.com/nf-core/ampliseq/pull/513#issuecomment-1375915648

Reply:
Perfect, thanks! :)